core/templates/pages/admin-page/misc-tab/admin-misc-tab.directive.ts

```diff
@@ -155,6 +157,22 @@ angular.module('oppia').directive('adminMiscTab', [
             });
         };
 
+        ctrl.getExplorationInteractions = function() {
+          $http.get(GET_INTERACTIONS_BY_ID_HANDLER_URL, {
+            params: {
+              exploration_id: ctrl.explorationId
+            }
+          })
+            .then(function(response) {
+              ctrl.explorationInteractionIds(
+                'Success! ' + response.data.interactions
+              );
+            }, function(errorResponse) {
+              ctrl.explorationInteractionIds(
+                'Server error: ' + errorResponse.data.error);
```

Comment:
Could we use the warning service for this, instead? See other similar errors that are shown elsewhere in the codebase.

Reply:
okay will take a look

Reply:
@BenHenning couldn't find what are you were referring here. There was no usage of warning service in this class either can you point where should i look?

Reply:
Ah, the admin page apparently does something different than the rest of the codebase. I suggest emulating the behavior here: https://github.com/oppia/oppia/blob/develop/core/templates/pages/admin-page/jobs-tab/admin-jobs-tab.directive.ts#L114. Basically, you can add another element to store your error and only show it when there's an error.

Reply:
okay will modify the code

Reply:
Done